Misc/NEWS.d/next/Library/2024-11-03-10-48-07.gh-issue-126353.ChDzot.rst

```diff
@@ -0,0 +1 @@
+:func:`asyncio.get_event_loop` now does not implicitly creates an event loop if not already created, instead it raises :exc:`RuntimeError`. Patch by Kumar Aditya.
```

Comment:
There may be something wrong with grammar here.

Reply:
improved it a bit